pkg/utils/utils_test.go

```diff
@@ -568,9 +1073,9 @@ func createAppDefinitionTags(app *appstacksv1beta1.RuntimeComponent) (map[string
 	}
 	return label, annotations
 }
-func createRuntimeComponent(n, ns string, spec appstacksv1beta1.RuntimeComponentSpec) *appstacksv1beta1.RuntimeComponent {
+func createRuntimeComponent(n, ns string, spec appstacksv1beta1.RuntimeComponentSpec, labels map[string]string, annotations map[string]string) *appstacksv1beta1.RuntimeComponent {
```

Comment:
It might be cleaner to have this take a `metav1.ObjectMeta` as a parameter instead of four separate params.